surch/constants.py

```diff
@@ -4,3 +4,9 @@
 DEFAULT_PATH = os.path.join(HOME_DIR, '.surch')
 CLONED_REPOS_PATH = os.path.join(DEFAULT_PATH, 'clones')
 RESULTS_PATH = os.path.join(DEFAULT_PATH, 'results')
+
+GITHUB_API_URL = 'https://api.github.com/{0}/{1}'
+REPO_DETAILS_API_URL = \
```

Comment:
- rename to GITHUB_REPO_DETAILS_API_URL to be more concise.
- how about defining this as ?GITHUB_REPO_DETAILS_API_URL = ''.join([GITHUB_API_URL, '/repos?type={2}&per_page={3}&page={4}'])